### ISSUE TITLE
Multiselect search string is passed to the change event when typing in the input

### ISSUE THREAD

**Issue by zdravkov** [telerik/kendo-vue#153]
Bug report: 
https://stackblitz.com/edit/bbuqh1?file=src/main.vue

Steps:
Focus the input, type some text and blur - 

Current:
an warning is observed in the console and the types string is passed to the change event.

Expected:
an warning is observed in the console and the types string SHOULD NOT BE passed to the change event.

**Comment by zdravkov** [comment]
temporary workaround - check if the value is  an array - https://stackblitz.com/edit/bbuqh1-he5s79?file=src/main.vue

```
  onChange (event)  {
             if (Array.isArray(event.target.value)) {
             this.value = [...event.target.value];
             }
        }
```

**Comment by zdravkov** [comment]
fixed in 3.3.2 - https://stackblitz.com/edit/bbuqh1-cwfwky?file=package.json,src%2Fmain.vue
